omod/pom.xml

```diff
@@ -1,49 +1,60 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-		xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-	<parent>
-		<groupId>org.openmrs.module</groupId>
-		<artifactId>ohricore</artifactId>
-		<version>1.0.0-SNAPSHOT</version>
-	</parent>
-
-	<artifactId>ohricore-omod</artifactId>
-	<packaging>jar</packaging>
-	<name>OHRICore OMOD</name>
-	<description>Omod submodule for OHRICore</description>
-
-	<dependencies>
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.openmrs.module</groupId>
+        <artifactId>ohricore</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>ohricore-omod</artifactId>
+    <packaging>jar</packaging>
+    <name>OHRICore OMOD</name>
+    <description>Omod submodule for OHRICore</description>
+
+    <dependencies>
+
         <dependency>
             <groupId>org.openmrs.module</groupId>
             <artifactId>ohricore-api</artifactId>
             <version>1.0.0-SNAPSHOT</version>
         </dependency>
-		<dependency>
-			<groupId>org.openmrs.web</groupId>
-			<artifactId>openmrs-web</artifactId>
+
+        <dependency>
+            <groupId>org.openmrs.web</groupId>
+            <artifactId>openmrs-web</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.openmrs.web</groupId>
+            <artifactId>openmrs-web</artifactId>
             <scope>provided</scope>
-		</dependency>
-		<dependency>
-			<groupId>org.openmrs.web</groupId>
-			<artifactId>openmrs-web</artifactId>
-			<scope>provided</scope>
-			<classifier>tests</classifier>
-		</dependency>
+            <classifier>tests</classifier>
+        </dependency>
+
+        <dependency>
+            <groupId>org.openmrs.module</groupId>
+            <artifactId>legacyui-omod</artifactId>
+            <version>1.8.2</version>
```

Comment:
@smallgod Do not hardcode versions in the internal pom.xml files, add the dependencies to the main pom and just reference them here

Reply:
moved this to the parent pom